src/commands/app/build.js

```diff
@@ -180,6 +180,11 @@ Build.flags = {
     default: true,
     allowNo: true
   }),
+  'web-optimize': flags.boolean({
+    description: '[default: false] Enable optimization (minification) of js/css/html',
+    default: false,
+    allowNo: true
```

Comment:
This line can be removed. I think we have been using `--no-` for those where default is true.

Reply:
good call, done!